exp/operator/go.mod

```diff
@@ -0,0 +1,24 @@
+module sigs.k8s.io/cluster-api/exp/operator
```

Comment:
What's the motivation behind having this as a separate module?

Reply:
This is done for better adoption of the new CAPI code. See: https://github.com/kubernetes-sigs/cluster-api/pull/4972. With a separate module, we can push some clusterctl code to master and avoid branch conflicts.